### PR TITLE
[New] Header menu refactoring

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.monday.com/boards/2218344956/pulses/2682163760

### CODE REVIEW

**Review by creed-victor — APPROVED** [review]

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx
Comment (creed-victor) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:528]: Was still able to click on this button when wallet is not connected (maybe because it become link when href was provided and not an actual button)

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx
Comment (tiltom) [reply src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:528]: Good catch, it looked disabled but that was just styling :D

**Review by creed-victor — APPROVED** [review]

**Comment by bitcoinuser** [comment]
On my screen, the Get started button is very near or Connect wallet button. And I think the buttons are very bug too, maybe can be smaller.

Captura de tela de 2022-05-20 11-20-33

**Review by soulBit — COMMENTED** [review]
Generally changes look good, but I've queried two points on the Story that will need to be clarified before I complete approval

File: src/app/components/Menu/components/MenuItem/index.tsx
Comment (pietro-maximoff) [inline src/app/components/Menu/components/MenuItem/index.tsx:10]: Since Icon component already merged into the dev branch i think make sense to replace FontAweasomeIcon with our Icon component

File: src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx
Comment (pietro-maximoff) [inline src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx:241]: replace pls data-action-id with dataActionId

Comment (tiltom) [reply src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx]: good catch, fixed :)

File: src/app/components/Menu/components/MenuItem/index.tsx
Comment (tiltom) [reply src/app/components/Menu/components/MenuItem/index.tsx]: Replaced, thanks for the suggestion.

**Comment by tiltom** [comment]
@soulBit's changes look good, I think it's ready to be merged :)

**Review by soulBit — APPROVED** [review]

**Review by pietro-maximoff — APPROVED** [review]

**Review by JoeStorm99 — APPROVED** [review]

**Pull request merged**
